### PR TITLE
A test case change is added to #13442: Fix  #12193 Servo displays upper level Thai character in wrong place. 

### LINKED ISSUE #12193: Servo displays upper level Thai character in wrong place.
On Ubuntu 16.04, Servo nightly, that I downloaded on 2016-07-03, displays upper level Thai characters in the incorrect place. Moreover it choose incorrect glyph too. 

Update: Servo on Mac OS X 10.11.5. has this issue too.

The text is "ยินดี". It is supposed to display    ี  on top of ด but it displays     ิ   on top of น, which is incorrect glyph and incorrect placement. 

servo_thai_bug

An URL for the test case is http://file.veerkesto.net/servo_test/simple-thai.html

### PR DESCRIPTION
**Pull request by veer66**
## A test case change is added to #13442

- [ ] `./mach build -d` does not report any errors
- [ ] `./mach test-tidy` does not report any errors
- [ ] These changes fix #__ (github issue number if applicable).

- [ ] There are tests for these changes OR
- [ ] These changes do not require tests because _____

---

This change is [Reviewable](https://reviewable.io/reviews/servo/servo/13506)

### CODE REVIEW

**Comment by jdm** [comment]
Rather than modifying the test, it would be better to duplicate it in tests/wpt/mozilla/tests/mozilla.

**Comment by veer66** [comment]
@jdm 

- Should I keep file name the same i.e. word-break-break-all-007.htm? 
- If I don't modify tests/wpt/css-tests/css-text-3_dev/html/word-break-break-all-007.htm, can this patch pass this test case?

**Comment by jdm** [comment]
* Yes
* No

**Comment by veer66** [comment]
@jdm So adding a new test case does not solve the issue in #13442?

**Comment by jdm** [comment]
The issue I'm interested in solving is a lack of tests that fail before the change and pass after it. It doesn't matter to me if the original test still fails, because we know why it fails. Does that make sense?

**Comment by veer66** [comment]
That sounds good.

**Comment by jdm** [comment]
Any progress, @veer66?

**Comment by KiChjang** [comment]
Looks like this is abandoned, needs a new owner.

**Comment by veer66** [comment]
There is no progress. I hope I can resume working on this in 2017.

**Comment by jdm** [comment]
```
  ▶ PASS [expected FAIL] /css-text-3_dev/html/word-break-break-all-007.htm
```
Woooooooo
This is totally ready to merge if word-break-break-all-007.htm.ini is removed.

File: tests/wpt/mozilla/tests/css/complex_glyphs_ref.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<!doctype html>
+<meta charset="utf-8">
+<body>
+ยิน&#X200Bดี
```
Comment (mbrubeck) [inline tests/wpt/mozilla/tests/css/complex_glyphs_ref.html]: There's a missing semicolon after `&#X200B;`

**Review by mbrubeck — CHANGES_REQUESTED** [review]

**Comment by mbrubeck** [comment]
r=mbrubeck with that minor bug in the test fixed.

**Pull request merged**